superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance/Options.h

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+struct Options {
```

Comment:
>Options [](http://example.com/codeflow?start=7&length=7)

there are lot of same functions as the class in another PR. How about we create a base function and create two derived for the two applications.

Reply:
ok, sure, updated to move the base class options.h to directx_utils, please check it